eclair-core/src/main/scala/fr/acinq/eclair/balance/BalanceActor.scala

```diff
@@ -0,0 +1,83 @@
+package fr.acinq.eclair.balance
+
+import akka.actor.typed.scaladsl.{ActorContext, Behaviors}
+import akka.actor.typed.{ActorRef, Behavior}
+import fr.acinq.eclair.balance.BalanceActor.{Command, TickBalance, WrappedChannels, WrappedGlobalBalance}
+import fr.acinq.eclair.balance.Monitoring.{Metrics, Tags}
+import fr.acinq.eclair.{Eclair, GlobalBalance}
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
+import scala.util.{Failure, Success, Try}
+
+object BalanceActor {
+
+  // @formatter:off
+  sealed trait Command
+  private final case object TickBalance extends Command
+  private final case class WrappedChannels(wrapped: ChannelsListener.GetChannelsResponse) extends Command
+  private final case class WrappedGlobalBalance(wrapped: Try[GlobalBalance]) extends Command
+  // @formatter:on
+
+  def apply(eclair: Eclair, channelsListener: ActorRef[ChannelsListener.GetChannels], interval: FiniteDuration)(implicit ec: ExecutionContext): Behavior[Command] = {
```

Comment:
As mentioned in another comment, I don't think actors should use the API, I think it should be the other way around: the `eclair: Eclair` API should be a thin wrapper around some logic implemented by actors.